src/main/resources/lib/ownership/cellWrapper.jelly

```diff
@@ -0,0 +1,20 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define">
+  <st:documentation>
+    The wrapper will be a `td` tag on Jenkins Core less than 2.264, and a `div` tag after that
```

Comment:
This comment is incorrect AFAICT. The summary pages div layout is applied only in particular pages. IIUC Node summaries have not been updated yet. Maybe @timja could clarify how it should work. IIUC the `divBasedFormLayout` should be automatic for Jenkins 2.271 for all Jelly layouts

Reply:
Right probably why this wasn’t updated at the time, it depends where this is displayed, if it’s wrapped in any layout by Jenkins core then this change may or may not be ok.

if it’s not wrapped in any table tags then it should be fine to change it without the bridging code

Reply:
I'm not quite sure what to do right now. Is OK to just remove the comment? I copied it from the `blockWrapper.jelly` file.
Or do you want me to change the implementation?

Reply:
@oleg-nenashev @timja any guidance to make this move forward ?

Reply:
If it fixes the bug then it should be fine

Reply:
I applied the suggested change to the docstring